mail/templates/product_order_receipt/body.html

```diff
@@ -0,0 +1,88 @@
+            </p>
+            {% for line in lines %}
+            <p>
+                <strong style="font-weight: 700;">Order Item:</strong> {{ line.content_title }}<br>
+                <strong style="font-weight: 700;">Product Number:</strong> {{ line.readable_id }}<br>
+                <strong style="font-weight: 700;">Dates:</strong>   {{ line.start_date|date:"F j, Y" }} - {{ line.end_date|date:"F j, Y" }}
+            </p>
+            <p>
+                <strong style="font-weight: 700;">Quantity:</strong>  {{ line.quantity }}<br>
+                <strong style="font-weight: 700;">Unit Price:</strong> ${{ line.price }}<br>
+                <strong style="font-weight: 700;">Discount:</strong> ${{ line.discount }}<br>
+                <strong style="font-weight: 700;">Total Paid:</strong> ${{ line.total_paid }}
+            </p>
+            {% endfor %}
+
+            <h3 style="color: #000000; font-size: 16px; font-weight: 700; line-height: 18px; margin: 0 0 20px;">Customer Information</h3>
+            <p style="margin: 0;">
+                <strong style="font-weight: 700;">Name:</strong> {{ purchaser.name }}<br>
+                <strong style="font-weight: 700;">Company Name:</strong> {{ purchaser.company }}<br>
+                {% if purchaser.street_address %}
+                    <table width="100%">
+                        <tr>
+                            <td style="width: 60px; vertical-align: top">
+                                <strong style="font-weight: 700;">Address:</strong>
+                            </td>
+                            <td>
+                                {% for address in purchaser.street_address %}
+                                    <div>{{ address }} </div>
+                                {% endfor %}
+                            <div>
+                                {{ purchaser.city}}, {{ purchaser.state_code}} {{ purchaser.postal_code}} {{ purchaser.country }}
+                            </div>
+                            </td>
+                        </tr>
+                    </table>
+                {% endif %}
+                <strong style="font-weight: 700;">Email Address:</strong> {{ purchaser.email }}<br>
+            </p>
+            {% if receipt or coupon %}
+                <h3 style="color: #000000; font-size: 16px; font-weight: 700; line-height: 18px; margin: 0 0 20px;">Payment Information</h3>
+                <p>
+                    {%  if receipt %}
+                        <strong style="font-weight: 700;">Name:</strong> {{ receipt.name }}<br>
+                        {% if receipt.payment_method == 'card' %}
+                            <strong style="font-weight: 700;">Payment Method:</strong> {{ receipt.card_type }} | {{ receipt.card_number }}<br>
+                        {% elif receipt.payment_method == 'paypal' %}
+                            <strong style="font-weight: 700;">Email:</strong> {{ receipt.email }} (Paypal account email)<br>
```

Comment:
Double-checked this against the CS test account and it does capture the email if they're regular credit card users. Prob makes sense to have the template differ here - I think that PayPal transactions pull the email from their PayPal account, rather than entering it like they do for credit cards.

On a related note: this template is 90% verbatim from xPRO so might be worth considering if this change needs to happen there too?